### PR TITLE
(13404) Add 'ha_cloud_instance_id' as a computed attribute in spoke and transit gateway resources

### PR DESCRIPTION
**Pull request by zhiqiangjin**

### CODE REVIEW

File: website/docs/r/aviatrix_spoke_gateway.html.markdown [hunk 1]
```diff
@@ -127,6 +127,7 @@ In addition to all arguments above, the following attributes are exported:
 * `security_group_id` - Security group used for the spoke gateway.
 * `cloud_instance_id` - Cloud instance ID of the spoke gateway.
 * `private_ip` - Private IP address of the spoke gateway created.
+* `ha_cloud_instance_id` - Cloud instance ID of the ha spoke gateway.
```
Comment (arvindsree) [inline website/docs/r/aviatrix_spoke_gateway.html.markdown]: HA in caps

File: website/docs/r/aviatrix_transit_gateway.html.markdown [hunk 2]
```diff
@@ -137,6 +137,7 @@ In addition to all arguments above, the following attributes are exported:
 * `security_group_id` - Security group used for the transit gateway.
 * `cloud_instance_id` - Cloud instance ID of the transit gateway.
 * `private_ip` - Private IP address of the transit gateway created.
+* `ha_cloud_instance_id` - Cloud instance ID of the ha transit gateway.
```
Comment (arvindsree) [inline website/docs/r/aviatrix_transit_gateway.html.markdown]: same

**Comment by arvindsree** [comment]
Add for regular gateway as well

File: website/docs/r/aviatrix_transit_gateway.html.markdown (hunk 2 above)
Comment (zhiqiangjin) [reply website/docs/r/aviatrix_transit_gateway.html.markdown]: All other ha based attributes are not in capital, like ha_subnet, ha_gw_size, and so on...

**Review by arvindsree — APPROVED** [review]

**Pull request merged**